examples/bpmn-utlis.js

```diff
@@ -0,0 +1,577 @@
+        <dc:Bounds x="670" y="266" width="88" height="68" />
+        <bpmndi:BPMNLabel />
+      </bpmndi:BPMNShape>
+      <bpmndi:BPMNShape id="Trisotech.Visio__6_end_event_2" bpmnElement="end_event_2">
+        <dc:Bounds x="804" y="284" width="32" height="32" />
+        <bpmndi:BPMNLabel>
+          <dc:Bounds x="796" y="325" width="48" height="27" />
+        </bpmndi:BPMNLabel>
+      </bpmndi:BPMNShape>
+      <bpmndi:BPMNShape id="Trisotech.Visio__6_task_4" bpmnElement="task_4">
+        <dc:Bounds x="545" y="369" width="88" height="68" />
+        <bpmndi:BPMNLabel />
+      </bpmndi:BPMNShape>
+      <bpmndi:BPMNShape id="Trisotech.Visio__6_task_5" bpmnElement="task_5">
+        <dc:Bounds x="681" y="369" width="83" height="68" />
+        <bpmndi:BPMNLabel />
+      </bpmndi:BPMNShape>
+      <bpmndi:BPMNShape id="Trisotech.Visio__6_end_event_3" bpmnElement="end_event_3">
+        <dc:Bounds x="816" y="387" width="32" height="32" />
+        <bpmndi:BPMNLabel>
+          <dc:Bounds x="793" y="432" width="77" height="14" />
+        </bpmndi:BPMNLabel>
+      </bpmndi:BPMNShape>
+      <bpmndi:BPMNShape id="Trisotech.Visio__6_end_event_1" bpmnElement="end_event_1">
+        <dc:Bounds x="890" y="108" width="32" height="32" />
+        <bpmndi:BPMNLabel>
+          <dc:Bounds x="861" y="153" width="100" height="14" />
+        </bpmndi:BPMNLabel>
+      </bpmndi:BPMNShape>
+      <bpmndi:BPMNShape id="Trisotech.Visio__6_task_1" bpmnElement="task_1">
+        <dc:Bounds x="238" y="90" width="83" height="68" />
+        <bpmndi:BPMNLabel />
+      </bpmndi:BPMNShape>
+      <bpmndi:BPMNShape id="Trisotech.Visio__6_start_event" bpmnElement="start_event">
+        <dc:Bounds x="155" y="109" width="30" height="30" />
+        <bpmndi:BPMNLabel>
+          <dc:Bounds x="133" y="153" width="73" height="14" />
+        </bpmndi:BPMNLabel>
+      </bpmndi:BPMNShape>
+      <bpmndi:BPMNShape id="Trisotech.Visio__6_sub_process" bpmnElement="sub_process" isExpanded="false">
+        <dc:Bounds x="750" y="90" width="83" height="68" />
+        <bpmndi:BPMNLabel>
+          <dc:Bounds x="760" y="100" width="63" height="48" />
+        </bpmndi:BPMNLabel>
+      </bpmndi:BPMNShape>
+      <bpmndi:BPMNShape id="Trisotech.Visio__6_boundary_event_2" bpmnElement="boundary_event_2">
+        <dc:Bounds x="470" y="234" width="32" height="32" />
+        <bpmndi:BPMNLabel>
+          <dc:Bounds x="402" y="266" width="68" height="14" />
+        </bpmndi:BPMNLabel>
+      </bpmndi:BPMNShape>
+      <bpmndi:BPMNShape id="Trisotech.Visio__6_boundary_event_1" bpmnElement="boundary_event_1">
+        <dc:Bounds x="538" y="234" width="32" height="32" />
+        <bpmndi:BPMNLabel>
+          <dc:Bounds x="573" y="266" width="63" height="14" />
+        </bpmndi:BPMNLabel>
+      </bpmndi:BPMNShape>
+    </bpmndi:BPMNPlane>
+  </bpmndi:BPMNDiagram>
+</semantic:definitions>
+        `;
+}
+
+function getProcurementBpmnDiagram() {
```

Comment:
:warning: missing source
:question: could we add it is intended to be used as call activity of the other one?

Reply:
The source is not missing is the same as the **getOrderFulfillmentBpmnDiagram()**. It's a block ;)

Reply:
We can use it as process called by call activity, but we can use it alone too ^^

Reply:
About the source, it wasn't clear this was a block (no end block) after so much lines.
Anyway, we could say `Same inspiration as for getOrderFulfillmentBpmnDiagram()`